app/frontend/util/ChampaignAPI.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import _ from 'lodash';
+import $ from 'jquery';
+import { camelizeKeys } from './util';
+
+export type OperationResponse = {
+  success: boolean,
+  errors: ?{[id:string]: string[]}
+}
+
+const parseResponse = (response):OperationResponse => {
+  if(response === undefined) {
+    return { success: true, errors: {} };
+  }
+
+  switch (response.status) {
+    case 200:
+    case 201:
+    case 204:
+      return { success: true, errors: {} };
+    case 422:
+      return { success: false, errors: camelizeKeys(response.responseJSON.errors) };
+    default:
+      return {
+        success: false,
+        errors: {
+          base: [<FormattedMessage id="call_tool.errors.unknown" defaultMessage={`Unknown error, code ${response.code}`} />]
+        }
+      };
+  }
+};
+
+
+const createCall = function(params: {pageId: string, memberPhoneNumber: string, targetIndex: string}) {
```

Comment:
If we export these functions directly we can import them like `import { createCalls } from 'path/to/api';`.

It's also idiomatic to namespace by file:

```js
import { createCall } from '../api/calls';
import { fetchBraintreeToken } from '../api/braintree';

// or...

import * as ChampaignAPI from '../api';
```

(but no need to change it).